### DISCUSSION TITLE
No placeholder text in input fields on checkout

### DISCUSSION THREAD

**Discussion by virajsoni06** [swellstores/community#251] [Help] [answered]
![image](https://github.com/swellstores/community/assets/13236286/74e3ffbd-6de5-46e5-b25f-af77233d1f50)
There are no placeholders on input fields in hosted checkout. Seems like a bug

**Comment by sags95** [comment]
Thanks for reporting this @virajsoni06, can you let me know what browser you're using? I'll be able to file this so the team can investigate.

**Comment by steven-k1** [comment, marked as answer]
@virajsoni06 We took a look at the checkout connected to your account and noticed that it had not been published as of yet, from what we can tell. We published the checkout, and we are seeing the placeholder text now on our end.

Would you be able to recheck on your end and confirm if the placeholder text is still missing for you on your end?

**Reply by virajsoni06** [reply]
Thanks! For future reference, how do I publish a checkout? Didn't find that option in Swell dashboard

**Comment by steven-k1** [comment]
@virajsoni06 Within the admin dashboard, navigate to storefronts and select 'Checkout' from the sub-options on the left. Once within the checkout page, select 'Customize checkout', and this will bring you to the checkout editor. In the top right of the page, you should see the option to 'Publish checkout'.

**Reply by virajsoni06** [reply]
Thank you
